code/style.css

```diff
@@ -1,7 +1,383 @@
+  cursor: pointer;
+}
+
+/* Accordion */
+
+.accordion {
+  display: block;
+  padding: 1rem;
+  width: 80%;
+  background-color: rgb(229, 241, 241);
+  color: #444;
+  transition: 0.4s;
+  border-radius: 2em 2em;
+  border: 2px solid black;
+  margin: 20px auto;
+}
+
+h1 {
+  color:rgb(243, 185, 118);
+  font-size: 1.8rem;
+}
+
+.question {
+  background-color: white;
+  cursor: pointer;
+  padding: 5px 5px 5px 5px;
+  margin-top: 10px;
+  border-radius: 1em 1em 1em 1em;
+  font-size: 22px;
+  transition: 0.4s;
+}
+
+.answer {
+  margin: 0;
+  padding: 0 10px;
+  background-color: white;
+  overflow: hidden;
+  transition: max-height 0.2s ease-out;
+  display: none;
+  border-radius: 1em 1em 1em 1em;
+  font-size: 18px;
+}
+
+.open {
+  background-color: rgb(229, 241, 241);
+}
+
+.open + .answer {
+  display: block;
+}
+
+.open + p {
+  display: block;
+}
+
+/* media query for desktop */
+
+@media screen and (min-width: 901px) {
+  .nav-toggle-label {
+    display: none;
+  }
+
+  header {
+    display: grid;
+    grid-template-columns: .5fr auto minmax(600px, 1fr);
+  }
+
+  nav {
+    color: black;
+    all:unset;
+    grid-column: 3/4;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+  }
+
+  nav ul {
+    display: flex;
+  }
+
+  nav li {
+    margin-right: 3em;
+    margin-bottom: 0;
+  }
+
+  nav a {
+    opacity: 1;
+    font-weight: 500;
+  }
+
+  .hero {
+    align-items: center;
+    justify-content: flex-end; 
+    flex-wrap: wrap;
+  }
+  .hero-content {
+    margin: 50px 80px 10px 10px;
+  }
+  
+  .hero h1, h2{
+    font-size: 40px;
+    color: rgb(245, 221, 195);
+  }
+  
+  .hero h1 {
+    margin-bottom: 0;
+    text-align: center;
+  }
+  
+  .hero h2 {
+    margin-top: 0;
+  }
+
+  .wrapper{
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    background-color: transparent;
+  }
+
+  .form {
+    grid-column: span 2;
+  }
+
+  .accordion {
+    grid-column: span 2;
+  }
+
+/* media query for tablet */
+
+@media screen and (max-width: 900px) and (min-width: 600px) {
+    .nav-toggle-label {
+      display: none;
+    }
+  
+    header {
+      display: grid;
+      grid-template-columns: .5fr auto minmax(600px, 1fr);
+    }
+  
+    nav {
+      color: black;
+      all:unset;
+      grid-column: 3/4;
+      display: flex;
+      justify-content: flex-end;
+      align-items: center;
+    }
+    nav ul {
+      display: flex;
+    }
+    nav li {
+      margin-right: 3em;
+      margin-bottom: 0;
+    }
+  
+    nav a {
+      opacity: 1;
+      font-weight: 500;
+    }
+  
+    .hero {
+      align-items: flex-end;
+      justify-content: flex-end; 
+      flex-wrap: wrap;
+    }
+    .hero-content {
+      margin: 10px 80px 10px 10px;
+    }
+    
+    .hero h1, h2{
+      font-size: 40px;
+      color: rgb(245, 221, 195);
+    }
+    
+    .hero h1 {
+      margin-bottom: 0;
+      text-align: center;
+    }
+    
+    .hero h2 {
+      margin-top: 0;
+    }
+  }
+}
```

Comment:
Your media queries for tablet and desktop are almost identical. I suggest, you get rid of all code that's double to make it easier to maintain and read 🙂

Reply:
Yes, I didn't have time to have a look at it in the end. I was suppose to fix it today.